readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -128,41 +110,53 @@ Promo.prototype.post_promo_display = function () {
 };
 
 function init() {
-    var post_data = {};
+    var get_data = {format: "jsonp"};
```

Comment:
Maybe just call this `data` or `request_data`, so it doesn't require changing based on get/post?